source/reactor/unit_tests/Main.cpp

```diff
@@ -0,0 +1,17 @@
+// Main.cpp : Defines the entry point for the console application.
+//
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+int main(int argc, char *argv[])
+{
+    ::testing::InitGoogleTest(&argc, argv);
+    ::testing::InitGoogleMock(&argc, argv);
+    int rc = RUN_ALL_TESTS();
+
+    return rc;

```

Comment:
You can use Google Test libraries in different ways: either you link against gtest_main or against gtest. In the first option there is a default `main()` function included. You use the second option when default `main()` doesn't suit your need. This is the case here because we need to init Google Mock as well.

Please see [Writing the main() Function](https://github.com/google/googletest/blob/master/googletest/docs/primer.md#writing-the-main-function)